### PR TITLE
Fix for width typo

### PR DESCRIPTION
**Pull request by jgordini**
changed from width to max-width in extra.css
Issue #202

### CODE REVIEW

**Comment by jgordini** [comment]
needs an additional fix.

**Comment by jgordini** [comment]
added max-width for mobile devices.

File: docs/uab_cloud/security_setup_basic.md [hunk 1]
```diff
@@ -90,35 +90,70 @@ Using a password protected Key Pair is highly recommended for additional securit
...
 
     1. Enter a "Key Pair Name".
+
     2. Select "SSH Key" in the "Key Type" drop down box.
 
         ![!Create Key pair dialog. The dialog form is filled out. The Key Pair Name is set to my_key_pair. ><](./images/key_pairs_002.png)
 
 5. Click "+ Create Key Pair"
 
+<<<<<<< HEAD
```
Comment (mdefende) [inline docs/uab_cloud/security_setup_basic.md:100]: Looks like there are still some merge conflict byproducts in this one.

File: docs/uab_cloud/security_setup_basic.md [hunk 2]
```diff
@@ -90,35 +90,70 @@ Using a password protected Key Pair is highly recommended for additional securit
...
 
     3. Redirects to the "Key Pairs" page.
     4. There should be a new entry in the table.
+>>>>>>> ca7117e354653a2a86a7d96fa9926c3abeb9053b
 
         ![!Key Pairs page. The Key Pairs table has one entry labeled my_key_pair. ><](./images/key_pairs_004.png)
 
 6. To add the private key on your local machine please see "Add key" under [Add a Private Key](./cloud_remote_access.md#add-a-private-key).
 
+<<<<<<< HEAD
```
Comment (mdefende) [inline docs/uab_cloud/security_setup_basic.md:126]: Merge conflict byproduct

File: docs/uab_cloud/security_setup_basic.md [hunk 3]
```diff
@@ -90,35 +90,70 @@ Using a password protected Key Pair is highly recommended for additional securit
...
 ### Generating a Key Pair Locally
+>>>>>>> ca7117e354653a2a86a7d96fa9926c3abeb9053b
 
 To generate a key pair, see instructions located at [Generating Key Pairs](./cloud_remote_access.md#generating-key-pairs).
 
 1. Click "Import Public Key" to open a dialog box.
 
 2. Fill out the dialog box.
 
+<<<<<<< HEAD
```
Comment (mdefende) [inline docs/uab_cloud/security_setup_basic.md:144]: Merge conflict byproduct

File: docs/uab_cloud/volume_setup_basic.md [hunk 4]
```diff
@@ -62,6 +62,11 @@ To format a volume, you must have created a volume and attached it to an instanc
 2. Scroll down to "Volumes Attached" and make note of the `<mount>` part of `<volume-name> on <mount>` for your attached volume as it will be used in later steps.
 
     ![!my_instance overview page. The page has been scrolled to the bottom. The mouse is pointing to a label under the Volumes Attached heading. The mouse is pointing to the Attached To label reading my_volume on /dev/vdb. ><](./images/persistent_volumes_000.png)
+<<<<<<< HEAD
```
Comment (mdefende) [inline docs/uab_cloud/volume_setup_basic.md:65]: Merge conflict byproduct. Looks like it was just the notation that wasn't removed

**Review by mdefende — CHANGES_REQUESTED** [review]
There are still a few sections of text from merge conflicts that need to be resolved on the security page.

**Comment by jgordini** [comment]
Crap. I'll take care of it.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jgordini** [comment]
Went in and removed the outdated head sections.

File: docs/uab_cloud/security_setup_basic.md [hunk 5]
```diff
@@ -119,6 +120,7 @@ To generate a key pair, see instructions located at [Generating Key Pairs](./clo
     1. Enter a "Key Pair Name".
     2. Select "SSH Key" in the "Key Type" drop-down box.
     3. Click "Browse..." to upload a public key file from your custom key pair **OR** copy-paste the content of that key file into the "Public Key" box.
+>>>>>>> ca7117e354653a2a86a7d96fa9926c3abeb9053b
```
Comment (mdefende) [inline docs/uab_cloud/security_setup_basic.md:123]: Can you also remove this commit hash?

**Comment by mdefende** [comment]
There's just one commit hash left in the security setup page. If you could remove that, we can go ahead and merge this.

**Review by mdefende — APPROVED** [review]

**Review by mdefende — APPROVED** [review]

**Pull request merged**